### PR TITLE
enhance: access log support print output fields of Query or Search

### PR DESCRIPTION
**Pull request by aoiasd**

### CODE REVIEW

**Comment by aoiasd** [comment]
rerun ut

File: internal/proxy/accesslog/info_test.go [hunk 1]
```diff
@@ -174,6 +174,18 @@ func (s *GrpcAccessInfoSuite) TestExpression() {
 	s.Equal(testExpr, result[0])
 }
 
+func (s *GrpcAccessInfoSuite) TestOutputFields() {
+	result := s.info.Get("$output_fields")
+	s.Equal(unknownString, result[0])
+
+	fileds := []string{"pk"}
```
Comment (congqixia) [inline internal/proxy/accesslog/info_test.go:181]: NIT: typo fileds => fields

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**